packages/babel-plugin-transform-react-jsx/README.md

```diff
@@ -9,16 +9,16 @@
 **In**
 
 ```javascript
-var profile = <div>
```

Comment:
So we are changing all the examples like this? May need to put somewhere that it's only in React 16, etc as well

Reply:
Should I split it into a second example (showcasing fragments specifically and noting it's React 16)?

Reply:
@clemmy yeah, let's split them up!